src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -505,6 +566,66 @@ describe('ThreadView', () => {
         .not
         .toBeInTheDocument();
     });
+
+    it('pressing load more button will load next page of comments', async () => {
+      await waitFor(() => renderComponent(discussionPostId));
+      await mockAxiosReturnPagedComments(discussionPostId, false, 2);
+
+      const loadMoreButton = await findLoadMoreCommentsButton();
+      fireEvent.click(loadMoreButton);
+
+      await screen.findByTestId('comment-comment-1');
+      await screen.findByTestId('comment-comment-4');
+    });
+
+    it('newly loaded comments are appended to the old ones', async () => {
+      await waitFor(() => renderComponent(discussionPostId));
```

Comment:
I think we don't need waitFor here. recheck it please
await renderComponent(discussionPostId);